src/compiler/symb-var.bas

```diff
@@ -157,6 +157,12 @@ function symbAddArrayDescriptorType _
 		aliasid &= dimensions
 	end if
 
+	'' top level const will be ignored in symbMangleType(), so use an alternate 
```

Comment:
It seems wrong to duplicate the code from `symbMangleType()`, and to mangle a part of the `arraydtype` outside the `<...>` part intended for it. But using somethin glike `symbMangleType( typeSetIsRef( arraydtype ) )`, like `symbMangleParam()`, would cause conflicts if FB ever supports arrays of references.

Maybe `symbMangleType()` needs an option to include the toplevel const for certain cases.

Reply:
OK.  I'm going to give symbMangleType() an option parameter.